.github/workflows/secrets-nats-kv.yml

```
@@ -45,3 +45,137 @@ jobs:
       - name: Run integration tests
         run: make test-integration-ci
         working-directory: ./crates/secrets-nats-kv
+
+  release-build:
+    if: ${{ startsWith(github.ref, 'refs/tags/secrets-nats-kv-v') }}
+    runs-on: ${{ matrix.build.runs-on }}
+    permissions:
+      contents: write
+    strategy:
+      matrix:
+        build:
+          - os: linux
+            arch: amd64
+            libc: gnu
+            runs-on: ubuntu-22.04
+            rust-target: x86_64-unknown-linux-gnu
```

```suggestion
          - os: linux
            arch: amd64
            libc: gnu
            runs-on: ubuntu-22.04
            rust-target: x86_64-unknown-linux-gnu
          - os: linux
            arch: aarch64
            libc: gnu
            runs-on: ubuntu-22.04
            rust-target: aarch64-unknown-linux-gnu
```

We need arm64 linux too right?

---

Ah thanks -- sure let me get get this building and update!

---

Done! Pipeline is working though we can't test the executables in GHA easily just yet.